gr-digital/python/digital/qa_pfb_clock_sync.py

```diff
@@ -165,8 +165,6 @@ def test03(self):
         self.tb.stop()
         self.tb.wait()
 
-        self.assertTrue(True)
```

Comment:
As far as I can tell, this is useless, so I removed it. (Same another instance below.)